app/views/pages/_home_page_bottom_section.html.erb

```diff
@@ -24,3 +13,40 @@
     </div>
   </div>
 </section>
+
+<section>
+  <div class="slab slab--white slab--padded slab--centered">
+    <div class="grid">
+      <div class="grid__item width-five-sixths shift-one-twelfth">
+        <p>
+          How it works
+        </p>
+        <p class="text--help">
+          MichiganBenefits.org is an easy, safe and secure way to apply for
+          Michigan's Food Assistance Program. Your information is securely submitted
+          as an official application with the Michigan Department of Health and
+          Human Services.
+        </p>
+        <p class="text--help">
+          For additional assistance email our
+          team at <a href="mailto:hello@michiganbenefits.org">hello@michiganbenefits.org</a>.
+        </p>
+        <p class="text--help">
+          To visit the official online application for food assistance, please visit
+          <%= link_to 'MI Bridges', 'https://www.michigan.gov/mibridges' %>.
+        </p>
+
+        <hr>
+
+        <p>
+          Private and Secure
+        </p>
+        <p class="text--help">
+          Your information is kept private and secure. MichiganBenefits
+          maintains strict security guidelines to protect your privacy and indentity.
+          For details, read the full <a href="/privacy" target="_blank">Privacy Policy</a>.
```

Comment:
`For details, read the full <%= link_to "Privacy Policy", privacy_path, target: "_blank" %>.`

Reply:
👍